src/harness/reference_models/dpa/move_list.py

```diff
@@ -477,11 +428,9 @@ def moveListConstraint(protection_point, low_freq, high_freq,
     grants = [cbsd_grants[0] for cbsd_grants in cbsds_grants_map.values()]
```

Comment:
Line 407: change "catb_neighbor_dist" to " neighbor_distances". Otherwise, we will get an error message in all the dpa examples and tests.

Reply:
Done. This was caused by a parallel commit on logging.